### ISSUE TITLE
NVDA does not announce/read dialog prompt information displayed in Win Live Mail.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#718]
**Reported by H.D. on 2010-06-18 23:11**
When deleting a message in Windows Live Mail,the NVDA (including latest snapshots) does not read the information displayed on the dialog prompt. The same issue is encountered when receiving a junk mail dialog prompt.

**Comment by leonardder** [comment]
I think this can be treated as similar windows live issues (i.e. won't fix).

**Comment by derekriemer** [comment]
Closing as wontfix
